packages/std/src/query/wasm.rs

```diff
@@ -37,3 +38,17 @@ pub struct ContractInfoResponse {
     /// set if this contract has bound an IBC port
     pub ibc_port: Option<String>,
 }
+
+/// Convenience constructor for tests / mocks
+#[doc(hidden)]
+impl ContractInfoResponse {
+    pub fn new(code_id: u64, creator: impl Into<String>) -> Self {
```

Comment:
Doesn't it belong here?

```suggestion
impl ContractInfoResponse {
    /// Convenience constructor for tests / mocks
    #[doc(hidden)]
    pub fn new(code_id: u64, creator: impl Into<String>) -> Self {
```